scene/2d/tile_map_layer.cpp

```diff
@@ -677,7 +677,7 @@ void TileMapLayer::_physics_update() {
 	const Ref<TileSet> &tile_set = get_effective_tile_set();
 
 	// Check if we should cleanup everything.
-	bool forced_cleanup = in_destructor || !enabled || !is_inside_tree() || !tile_set.is_valid();
+	bool forced_cleanup = in_destructor || !enabled || is_inside_tree() || !tile_set.is_valid() || !tile_map_node->is_enabled();
```

Comment:
```suggestion
	bool forced_cleanup = in_destructor || !enabled || !is_inside_tree() || !tile_set.is_valid() || !tile_map_node->is_enabled();
```
Unless I'm mistaken this is necessary, cc @groud

Reply:
sorry. my fail.